skbio/alignment/_tabular_msa.py

```diff
@@ -201,6 +201,16 @@ def keys(self):
 
     @property
     @experimental(as_of='0.4.0-dev')
+    def minter(self):
```

Comment:
@jairideout is this going to supercede `has_keys`? They seem to pretty much mean the same thing.